types/flatbuffers/index.d.ts

```diff
@@ -0,0 +1,610 @@
+// Type definitions for flatbuffers 1.6
+// Project: http://google.github.io/flatbuffers/index.html
+// Definitions by: Kamil Rojewski <kamil.rojewski@gmail.com>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare module flatbuffers {
```

Comment:
Since this is namespace, not an ES6 module, it would be better to use the `declare namespace` syntax instead of the legacy `declare module` syntax.